src/fmu/dataio/datastructure/meta/meta.py

```diff
@@ -341,23 +398,25 @@ class Context(BaseModel):
     stage: enums.FmuContext
 
 
-class FMUClassMetaCase(BaseModel):
+class FMUCaseAttributes(BaseModel):
```

Comment:
An exception about the `FMU` prefix is here, because we are speaking explicitly about the `fmu: ` block in the data model.

Would appreciate feedback for possibly better names for this class